### DISCUSSION TITLE
UpdatesAtStartup missing

### DISCUSSION THREAD

**Discussion by s016aq** [Romanitho/Winget-AutoUpdate#383] [General]
Is there a reason that the option to trigger updates on startup is removed?
It worked in 1.16.2 and is missing in 1.17.7
I used this on our AVDs and now it does not work when trying to upgrade to 1.17.7

**Comment by s016aq** [comment]
Sorry. Ignore this. It was I who added it to the script. The dementia is coming on strong!

**Reply by KnifMelti** [reply]
This you now can do via #386
